charts/stardog/files/utils.sh

```diff
@@ -18,8 +18,6 @@ function wait_for_start {
       curl -v  http://${HOST}:${PORT}/admin/healthcheck
       RC=$?
     done
-    # Give it a second to finish starting up
-    sleep 20
```

Comment:
are we sure it is actually ready to go when the health check clears here?  I think that this will be true even if the kernel has not fully booted :-/

Reply:
lets just note this and fix it properly in the cluster if it comes it.